spinnaker_camera_driver/scripts/linux_setup_flir.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+echo ""
+echo -e "\e[32mStarting spinnaker_camera_driver setup script\e[0m"
+echo ""
+
+sudo addgroup flirimaging
```

Comment:
Add a prompt to ask if they would like do this section.

Reply:
You should expect people to run the script even though the spinnaker SDK has already been installed. In fact I'd urge you to verify that nothing bad happens if you run the script in that scenario (or run it twice).
Here is how to check if the group already exists:

https://stackoverflow.com/questions/29073210/how-to-check-if-a-group-exists-and-add-if-it-doesnt-in-linux-shell-script

Reply:
Thanks @berndpfrommer for the feedback. I didn't add a check to see if the group already exists because I have run it repeatedly with no consequenses (it just output that the group already existed), I agree though that it would be better form to explicitly check.